search/searcher/search_ip_range.go

```diff
@@ -0,0 +1,67 @@
+//  Copyright (c) 2014 Couchbase, Inc.
```

Comment:
2021*

Reply:
cool...
@abhinavdangeti
Which copyright should I copy>

Reply:
@amnonbc  This is the bit (from search/searcher/search_ip_range.go), that's missing in some files ..

```
//  Copyright (c) 2021 Couchbase, Inc.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
// 		http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```
- document/field_ip.go
- document/field_ip_test.go
- search/query/ip_range.go 
- search/searcher/search_ip_range_test.go
- test/ip_field_test.go

Reply:
OK, added.

Reply:
I get a unit test failure when running against 1.17.
This will need a bit of work and some coffee to figure out.
I'll do it tomorrow. (It is nearly 11pm in London now).

Reply:
OK. I had some coffee and fixed the breaking test.
The standard library in Go 1.17 has become stricter on parsing IP addresses, and was rejected IP input which 
contained gratuitions leading zeros.